specification/cpim/resource-manager/Microsoft.AzureActiveDirectory/stable/2021-04-01/externalIdentities.json

```diff
@@ -0,0 +1,1311 @@
+          "final-state-via": "location"
+        }
+      }
+    },
+    "/providers/Microsoft.AzureActiveDirectory/operations": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_List",
+        "x-ms-examples": {
+          "operations_list": {
+            "$ref": "./examples/OperationsList.json"
+          }
+        },
+        "description": "Lists the operations available from this provider.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Describe the result of a successful operation.",
+            "schema": {
+              "$ref": "#/definitions/AvailableOperations"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-pageable": {
+          "nextLinkName": null
+        }
+      }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.AzureActiveDirectory/operations/{operationId}": {
+      "get": {
+        "tags": [
+          "Operations"
+        ],
+        "operationId": "Operations_GetAsyncStatus",
+        "description": "Gets the status of the async operation.",
+        "x-ms-examples": {
+          "Get Async Status": {
+            "$ref": "./examples/getAsyncOperationStatus.json"
+          }
+        },
+        "x-ms-long-running-operation": true,
+        "x-ms-long-running-operation-options": {
+          "final-state-via": "location"
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
+          },
+          {
+            "$ref": "#/parameters/ApiVersionParameter"
+          },
+          {
+            "name": "operationId",
+            "type": "string",
+            "in": "path",
+            "description": "The operation ID.",
+            "required": true
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "The async operation status.",
+            "schema": {
+              "$ref": "#/definitions/AsyncOperationStatus"
+            }
+          },
+          "202": {
+            "description": "Tenant create request accepted. The Location header indicates the URL/location in which to check to get the async status of the operation.",
+            "headers": {
+              "Location": {
+                "description": "Contains the URL/location which can return the tenant provisioning status.",
+                "type": "string"
+              },
+              "Retry-After": {
+                "description": "Contains the number of seconds to wait before checking the tenant provisioning status.",
+                "type": "string"
+              }
```

Comment:
If this api is for Azure-AsyncOperation header, it should always return 200.

If you are using Location headers only for async operation, please follow the RPC and we don't need to define operationResults api in the swagger.
[https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers](https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/async-api-reference.md#202-accepted-and-location-headers)